hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/staking/StakingSuite.java

```diff
@@ -68,12 +67,10 @@
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.MethodOrderer;
 import org.junit.jupiter.api.Order;
-import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.TestMethodOrder;
 
 @HapiTestSuite
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
-@Tag(TIME_CONSUMING)
```

Comment:
Was this intentionally removed? Moving `StakingSuite` into `MISC` (default category) instead of `TIME_CONSUMING` means that `Misc` check takes ~80 minutes instead of ~50; while `TIME_CONSUMING` goes from ~60min to ~30min.

Reply:
@jjohannes Please see if we can mitigate the concerns @tinker-michaelj raised here.